lib/index.js

```diff
@@ -0,0 +1,144 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const _constants = require('./constants');
+const bedrock = require('bedrock');
+const brJobs = require('bedrock-jobs');
+const cache = require('bedrock-redis');
+const {config, util: {clone}} = bedrock;
+const assert = require('assert-plus');
+const keys = require('./keys');
+
+require('./config');
+const cfg = config['stats-storage-redis'];
+
+bedrock.events.on('bedrock.start', async () => {
+  const jobQueue = exports._jobQueue = brJobs.addQueue(
+    {name: _constants.JOB_QUEUE});
+  // setup a processor for the queue with the default concurrency of 1
+  jobQueue.process(async () => {
+    const beforeDate = Date.now() - cfg.history.ttl;
+    return exports.trimHistory({beforeDate});
+  });
+  await jobQueue.add({}, {
+    // prevent duplicate jobs by specifying a non-unique jobId
+    jobId: 'trimHistory',
+    // repeated jobs are completed and rescheduled on every iteration
+    repeat: {every: cfg.history.trimInterval},
+    // do not keep record of successfully completed jobs in redis
+    removeOnComplete: true
+  });
+});
+
+/**
+ * @module bedrock-stats-storage-redis
+ */
+
+exports.keys = keys;
+
+/**
+ * @description Query stats history.
```

Comment:
We're relaxing that jsdoc rule, so these descriptions could be changed back to regular text.

Reply:
done